public/locales/en/common.json

```diff
@@ -177,11 +180,12 @@
   "privacyLinkText": "Review the privacy policy",
   "becomeAParticipantDescription": "You’re invited to test our experimental ideas and participate in research interviews to make Service Canada better for everyone. Every bit of feedback helps us make sure we’re making our services simple and easy to use.",
   "thankYou": "Thank you!",
-  "confirmationP1": "We’ve added you to our list and sent you a confirmation email from",
+  "confirmationP1": "We have sent a confirmation email to <<youremail@email.com>> from",
```

Comment:
Just a question, but are we expected to replace `<<youremail@email.com>>` with the actual email address from the user?

Reply:
I'm not sure actually.